<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FluidMaterial</class>
    <widget class="QWidget" name="FluidMaterial">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>338</width>
                <height>220</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Mesh Import</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="0" column="0" colspan="2">
                            <widget class="QLabel" name="label">
                                <property name="font">
                                    <font>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Mesh to import</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QPushButton" name="pb_select_mesh">
                                <property name="text">
                                    <string>Select Mesh</string>
                                </property>
                            </widget>
                        </item>
						<item row="1" column="1">
                            <widget class="QPushButton" name="pb_clear_selection">
                                <property name="text">
                                    <string>Clear</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0" colspan="2">
                            <widget class="QLabel" name="labelInputMeshFilename">
                                <property name="font">
                                    <font>
                                        <bold>false</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="1" column="0" colspan="2">
                            <widget class="QPushButton" name="pb_do_import">
                                <property name="text">
                                    <string>Import mesh ...</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" name="gridLayout_2">
                    <property name="leftMargin">
                        <number>8</number>
                    </property>
                    <property name="topMargin">
                        <number>8</number>
                    </property>
                    <property name="rightMargin">
                        <number>8</number>
                    </property>
                    <property name="bottomMargin">
                        <number>8</number>
                    </property>
                    <property name="spacing">
                        <number>8</number>
                    </property>
                    <item row="0" column="0">
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Status</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <layout class="QGridLayout" name="gl_actions">
                            <item row="1" column="0">
                                <widget class="QLabel" name="l_time">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="0">
                                <widget class="QTextEdit" name="te_output">
                                    <property name="lineWrapMode">
                                        <enum>QTextEdit::NoWrap</enum>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
